server/src/user/entities/onetime-invite-request.entity.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { User } from './user.entity';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import { Allow, IsOptional } from 'class-validator';
+import { CreateDateColumnTz } from 'src/datasources/basecolumns';
+import { Community } from './community.entity';
+import { Ty } from 'src/tasks/entities/type';
+import { Notification } from 'src/notifs/entities/notification.entity';
+
+export enum OnetimeInviteRequestStatus {
+  PENDING = 'pending',
+  APPROVED = 'approved',
+  REJECTED = 'rejected',
+}
+
+@Entity()
+export class OnetimeInviteRequest {
```

Comment:
im confused why we want two entities here? my impression is that this entity is very duplicative of the existing `OnetimeInvite` entity, just with an extra status field. instead of having 2 entities that are almost the same, is there a reason that we couldn't have just added a requestStatus to the existing entity, and have both invite processes create the same entity (that just defaults to approved for leaders)?